app/views/incomes/EditSuccessView.scala.html

```diff
@@ -32,8 +32,7 @@
     title = messages("tai.incomes.updated.check.title"),
     authedUser = Some(user),
     pageTitle = Some(Messages("tai.service.navTitle")),
-    backLinkContent = Some(Messages("tai.back-link.upper")),
-    backLinkId = "backLink",
+    backLinkContent = None,
```

Comment:
why is backLinkContent None, out of interest?

Reply:
So the Success View in other views has never had the back link. I looked at it and it was introduced recently but I don't think it should be there

Reply:
Thank you!